tensorflow/lite/micro/system_setup.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+// This is part of the ultra-lightweight testing framework designed for use with
+// microcontroller applications. See tensorflow/lite/micro/testing/micro_test.h.
+
+#ifndef TENSORFLOW_LITE_MICRO_TESTING_SYSTEM_SETUP_H_
+#define TENSORFLOW_LITE_MICRO_TESTING_SYSTEM_SETUP_H_
+
+namespace system_setup {
```

Comment:
nit: we're going with a flat tflite namespace and descriptive function names.

If you'd prefer we can go with tflite::InitializeTarget instead of tflite::Initialize.

Reply:
here's some more context on this decision: https://abseil.io/tips/130

Reply:
Sure